config/resources/auth.json

```diff
@@ -0,0 +1,73 @@
+{
+    "version": "0.1",
+    "prefixes": {
+        "foaf": "http://xmlns.com/foaf/0.1/",
+        "besluit": "http://data.vlaanderen.be/ns/besluit#",
+        "dct": "http://purl.org/dc/terms/",
+        "skos": "http://www.w3.org/2004/02/skos/core#"
+    },
+    "resources": {
+        "users": {
```

Comment:
This doesn't require a match config in the dispatcher?

Reply:
not for now (the mock-login doesn't require it) but might be needed if we do a specific page to show the user's information at some point.